### PR TITLE
Add more tests in tf_trt_integration_test.py.

### PR DESCRIPTION
**Pull request by aaroey**
Changes include:
- Consolidate tf_trt_integration_test.py
- Remove test_tftrt.py
- Fix a resource leaking issue when converting calibration graph to inference graph

### CODE REVIEW

File: tensorflow/contrib/tensorrt/test/test_tftrt.py [hunk 1]
```diff
@@ -1,287 +0,0 @@
-# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```
Comment (samikama) [inline tensorflow/contrib/tensorrt/test/test_tftrt.py]: Pleas keep this file

File: tensorflow/contrib/tensorrt/test/tf_trt_integration_test.py [hunk 2]
```diff
@@ -18,131 +18,332 @@
...
-
-  def get_simple_graph_def(self):
-    """Create a simple graph and return its graph_def."""
-    g = ops.Graph()
-    with g.as_default():
-      a = aops.placeholder(
-          dtype=dtypes.float32, shape=(None, 24, 24, 2), name="input")
-      e = cop.constant(
+
+def GetSingleEngineGraphDef():
```
Comment (samikama) [inline tensorflow/contrib/tensorrt/test/tf_trt_integration_test.py]: Adding a datatype option might be a good idea

**Comment by samikama** [comment]
Very good @aaroey, Thanks for the changes and sorry for delaying it so long, forcing you do it.

**Review by samikama — DISMISSED** [review]

File: tensorflow/contrib/tensorrt/test/test_tftrt.py (hunk 1 above)
Comment (aaroey) [reply tensorflow/contrib/tensorrt/test/test_tftrt.py]: Done.

File: tensorflow/contrib/tensorrt/test/tf_trt_integration_test.py (hunk 2 above)
Comment (aaroey) [reply tensorflow/contrib/tensorrt/test/tf_trt_integration_test.py]: Done.

**Review by samikama — APPROVED** [review]

**Pull request merged**